### PR TITLE
Removed upgrade steps while developing 6->7

### PR DESCRIPTION
**Pull request by abel-navarro**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1102,6 +1102,15 @@ function cloudupgrade()
         # upgrade and reapply all barclamps
         onadmin runlist crowbar_nodeupgrade \
                         proposal
+
+    else if iscloudver 6plus; then
```
Comment (jsuchome) [inline scripts/mkcloud]: You have to put it above, `iscloudver 5plus` is true for Cloud6

**Comment by jsuchome** [comment]
This basically makes the CI job https://ci.suse.de/view/Cloud/view/Cloud7/job/cloud-mkcloud7-job-upgrade-x86_64/ not usable. Well, usable, but with different results that we expected: only admin server would be installed.

Also, it makes https://github.com/crowbar/crowbar-core/pull/651 unnecessary.

That said, I do not mind going this way

File: scripts/mkcloud [hunk 2]
```diff
@@ -1097,11 +1097,20 @@ function crowbarupgrade_5plus()
...
-    if iscloudver 5plus ; then
+    if iscloudver 6plus ; then
+        crowbarupgrade_5plus
+        # upgrade and reapply all barclamps
+
+        # commented out while developing the 6->7 upgrade process
+        #onadmin runlist crowbar_nodeupgrade \
+        #                proposal
+
+    elif iscloudver 5plus; then
```
Comment (jsuchome) [inline scripts/mkcloud]: `iscloudver 5` would be enough, btw

**Comment by jsuchome** [comment]
Actuall, I agree with this. We do not have ways to reinstalling nodes now (https://github.com/crowbar/crowbar-core/pull/651 is not enough. ) 
We could revert `crowbar_service.rb` part of https://github.com/crowbar/crowbar-core/pull/640, but I do not see a reason why continue with testing upgrade of nodes when it is not implemented yet.

+1

**Comment by jsuchome** [comment]
@vuntz , what do you think?

**Comment by jsuchome** [comment]
@abel-navarro I may have alternative proposal:

1. leave mkcloud upgrade step as it is, that should make https://ci.suse.de/view/Cloud/view/Cloud7/job/cloud-mkcloud7-job-upgrade-x86_64/ green

2. for non-disruptive ugrade, we need extra jobs anyway (see https://trello.com/c/BfMFXB3r/64-5-s47p5-s48p19-add-ci-jobs-for-new-upgrade-process-with-and-without-ha). Here I would set up one or two new jobs that would not use `cloudupgrade` command, but something like `cloud6upgrade` which would basically be this PR of yours: `cloudupgrade` witht the nodes stuff commented.

3. We will be gradually adapting that new command by adding new stuff step by step as being implemented

**Comment by abel-navarro** [comment]
@jsuchome I like that proposal. Today I cannot work on it but can retake it asap.

**Comment by abel-navarro** [comment]
Closing this PR as it will be overridden by either #1234 or #1237.

**Pull request closed without merging**